config/service_providers.yml

```diff
@@ -397,3 +397,12 @@ production:
     logo: 'cbp-ttp.png'
     redirect_uris:
       - 'https://ttp.cbp.dhs.gov'
+
+  # CBP OARS
+  'urn:gov:dhs.cbp.pspd.oars:openidconnect:prod:app':
+    friendly_name: 'CBP OARS'
+    agency: 'DHS'
+    logo: 'cbp.png'
+    redirect_uris:
+      - 'gov.dhs.cbp.pspd.oars.user.prod//result'
```

Comment:
Hi I just spotted that this does not contain a colon, so it's probably not a valid URL.

Should this be:

```diff
-gov.dhs.cbp.pspd.oars.user.prod//result
+gov.dhs.cbp.pspd.oars.user.prod://result
```

?